diff --git a/node_modules/darkreader/darkreader.js b/node_modules/darkreader/darkreader.js
index 71939ba..65615fc 100644
--- a/node_modules/darkreader/darkreader.js
+++ b/node_modules/darkreader/darkreader.js
@@ -4856,7 +4856,7 @@
         var rootVarsStyle = createOrUpdateStyle('darkreader--root-vars');
         document.head.insertBefore(rootVarsStyle, variableStyle.nextSibling);
         var proxyScript = createOrUpdateScript('darkreader--proxy');
-        proxyScript.append("(" + injectProxy + ")(!" + fixes.disableStyleSheetsProxy + ")");
+        proxyScript.append("(" + injectProxy + ")");
         document.head.insertBefore(proxyScript, rootVarsStyle.nextSibling);
         proxyScript.remove();
     }
